xml/security_pcidss.xml

```diff
@@ -7,272 +7,1251 @@
+       login time recorded in <filename>/var/log/lastlog</filename> and
+       calculates the number of days since.
+       By default, access is denied when the inactivity reaches 90 days.
+      </para>
+      <para>
+       To list the local accounts last login time use the command
+       <command>lastlog</command>.
+      </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry xml:id="vle.pcidss.lock-user">
+      <term>
+       8.1.6 Limit repeated access attempts by locking out the user ID after
+       not more than six attempts.
+      </term>
+      <listitem>
+       <para>
+        As stated in <xref linkend="vle.pcidss.disable-user"/>, a centralized
+        account infrastructure will have this capability. On the
+        &productname; system, we can check and limit access attempts with the
+        <systemitem>pam_tally2</systemitem> PAM module. The module is
+        executed during login time and checks the recorded failed attempts
+        since the last successful login. To check and reset the account
+        status, use the tool <command>pam_tally2</command>.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>
+       8.1.7 Set the lockout duration to a minimum of 30 minutes or until an
+       administrator enables the user ID.
+      </term>
+      <listitem>
+      <para>
+       The PAM module <systemitem>pam_tally2</systemitem> that was described
+       for <xref linkend="vle.pcidss.lock-user"/> can be used to lock an
+       account for a given time after a failed login attempt. The parameter
+       <literal>unlock_time=1800</literal> must be configured into the PAM
+       configuration. The default setting is that only the administrator can
+       reactivate a locked account.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry xml:id="vle.pcidss.multi-factor-local">
+      <term>
+       8.3.1 Incorporate multi-factor authentication for all non-console access
+       into the CDE for personnel with administrative access.
+      </term>
+      <listitem>
+       <para>
+        To authenticate users for administrative access with multiple factors, use
+        the pluggable authentication module (PAM) mechanism. This makes it
+        flexible to add new methods to the authentication process and to
+        adjust it if needed.<!--fixme wording-->
+       </para>
+       <para>
+        Another way of multi-factor authentication for SSH connections is the
+        additional usage of public keys. To connect to a system, it is
+        necessary to prove that you are in the possession of the appropriate
+        private key. When accepted you need to enter a password at the second
+        stage. Thus, attackers need to acquire a private key before they can
+        even try to brute force a password prompt.
+       </para>
+       <para>
+        For commercial one-time password (OTP) products, there is usually
```

Comment:
<del>For commercial one-time password (OTP) products, there is usually also a Linux PAM module available.</del> A Linux PAM module is usually available for commercial one-time password (OTP) products.